src/index.ts

```diff
@@ -92,7 +93,7 @@ fixtures.browserOptions.init(async ({ headful, slowMo }, run) => {
   await run({
     handleSIGINT: false,
     slowMo,
-    headless: !headful,
+    headless: headful === undefined ? undefined : !headful,
```

Comment:
```suggestion
    ...(typeof headful !== "undefined" && { headless: !headful })
```
I think that's cleaner, but open for discussion.

Reply:
My thoughts are that a ternary operator is simpler to understand.

Reply:
and it destructuring would crash if the condition is not met `...(false)`

Reply:
I don't think that last point is true. https://repl.it/@nrhoffmann/PeachpuffWindingOutput#index.ts